modules/cnf-image-based-upgrade-prep-resources.adoc

```diff
@@ -0,0 +1,198 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-image-based-upgrade-creating-backup-resources_{context}"]
+= Creating ConfigMap objects for the image-based upgrade with {lcao}
+
+The {lcao} needs all your OADP resources, extra manifests, and custom catalog sources wrapped in a `ConfigMap` object to process them for the image-based upgrade.
+
+
+[id="cnf-image-based-upgrade-creating-backup-oadp-resources_{context}"]
+== Creating OADP ConfigMap objects for the image-based upgrade with {lcao}
+
+Create your OADP resources that are used to back up and restore your resources during the upgrade.
+
+.Prerequisites
+
+* Generate a seed image from a compatible seed cluster.
+* Create backup and restore resources.
+* Create a separate partition on the target cluster for the container images that is shared between stateroots. For more information about, see _Additional resources_.
+* Deploy a version of {lcao} that is compatible with the version used with the seed image.
+* Install the OADP Operator, the `DataProtectionApplication` CR and its secret on the target cluster.
+* Create an S3-compatible storage solution and a ready-to-use bucket with proper credentials configured. For more information, see _Additional resources_.
+
+.Procedure
+
+. Create your OADP `Backup` and `Restore` CRs in the same namespace where the OADP Operator is installed, which is `openshift-adp`.
++
+--
+include::snippets/ibu-PlatformBackupRestore.adoc[]
+
+[source,yaml]
+----
+include::snippets/ibu-ApplicationBackupRestore.yaml[]
+----
+
+[NOTE]
+====
+Depending on your {rh-rhacm} configuration, you might need to include the `v1/secrets/open-cluster-management-agent/open-cluster-management-image-pull-credentials` object for backup.
+If your `multiclusterHub` CR has `.spec.imagePullSecret` defined and the secret exists on the `open-cluster-management-agent` namespace in your hub cluster, ensure that the object is included in the `lca.openshift.io/apply-label` annotation. If the secret does not exist, you can remove the object from the apply-label annotation.
```

Comment:
I would move this to the ibu-PlatformBackupRestore.yaml. See the comment in that file
```suggestion
```